src/sentry/integrations/vsts/webhooks.py

```diff
@@ -62,6 +70,24 @@ def handle_assign_to(self, integration, external_issue_key, assigned_to):
             assign=assign,
         )
 
+    def handle_status_change(self, integration, external_issue_key,
+                             status_change, project):
+        if status_change is None:
+            return
+
+        organization_ids = OrganizationIntegration.objects.filter(
+            integration_id=integration.id,
+        ).values_list('organization_id', flat=True)
+
+        for organization_id in organization_ids:
+            installation = integration.get_installation(organization_id)
+            data = {
+                'new_state': status_change['newValue'],
+                'old_state': status_change['oldValue'],
+                'project': project,
+            }
+            installation.sync_status_inbound(external_issue_key, data)
```

Comment:
@macqueen could adding `kwargs` to `sync_status_inbound` be a better answer or is that ok?